.gitignore

```diff
@@ -19,3 +19,7 @@
 
 *.DS_Store
 
+
+config/initializers/ios_certificate.rb
+
+config/initializers/ios_development_certificate.rb
```

Comment:
Is this right? Seems strange to store a certificate in a Ruby file. How does the production Ruby file get into production?